src/plugins/data/public/query/filter_manager/filter_manager.ts

```diff
@@ -221,4 +229,17 @@ export class FilterManager {
       }
     });
   }
+
+  // Filter needs to implement SerializableState
+  public extract = extract as any;
```

Comment:
If it needs to be `SerializableState`, maybe you can remove the `any`?

```suggestion
  public extract = extract as SerializableState;
```

Reply:
i can't as Filters don't match the SerializableState definition yet

Reply:
https://github.com/elastic/kibana/issues/107365